src/generators/app/templates/src/shared/api/http/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint no-confusing-arrow: 0 */
```

Comment:
if you disable something, you should enable it at the end of the file.

Otherwise, you should disable it globally in eslintrc.